director.css

```diff
@@ -7,6 +7,11 @@ select, div {
   margin: 20px;
 }
 
+.ui-menu .ui-state-focus,
+.ui-menu .ui-state-active {
+  margin: 20px ! important;
```

Comment:
thank you for this, I was bothered by it but couldn't really figure it out after a few minutes